meson_options.txt

```diff
@@ -298,6 +298,8 @@ option('slow-tests', type : 'boolean', value : 'false',
 option('install-tests', type : 'boolean', value : 'false',
        description : 'install test executables')
 
+option('ok-color', type: 'combo', choices : ['blue', 'green', 'orange'],
+       description: 'color of the "OK" status message')
```

Comment:
This changes the default to blue. Please put 'green' first so that default is not changed.

Reply:
Added `value : 'green'` for setting green as the default value, to keep the colors in the same order as the ANSI color numbers.